### PR TITLE
adds envio to polygon docs as supported data indexer

### PR DESCRIPTION
**Pull request by SvenMuller95**
We are a supported data indexing framework for Polygon PoS, zkEVM, and CDK and on the Polygon Solutions Provider Network.

### CODE REVIEW

**Review by rahuldamodar94 — APPROVED** [review]

**Comment by Danyalkasiri** [comment]
[](url)

**Comment by Danyalkasiri** [comment]
Diff

**Comment by Danyalkasiri** [comment]
1. > [__](url)

**Comment by Danyalkasiri** [comment]
Ko

**Comment by Danyalkasiri** [comment]
Polygon PoS، zkEV

**Comment by Danyalkasiri** [comment]
#296

**Comment by kmurphypolygon** [comment]
Converting to draft for copy edit.

@SvenMuller95 if you could do a quick pass on headers (sentence case) and extra lines as per https://learn.microsoft.com/en-us/style-guide/welcome/ that would be very helpful. I'll add 2 suggestions so you can see what I mean.

File: docs/tools/data/envio.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+!!! note "Content disclaimer"
+
+    Please view the third-party content disclaimer [here](https://github.com/0xPolygon/polygon-docs/blob/main/CONTENT_DISCLAIMER.md).
+
```
Comment (kmurphypolygon) [inline docs/tools/data/envio.md:4]:
```suggestion
```

File: docs/tools/data/envio.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+## Envio HyperSync 
+
+Envio supports [HyperSync](https://docs.envio.dev/docs/hypersync) on Polygon networks. 
+
+HyperSync is an accelerated data query layer for the Polygon networks, providing APIs that bypass JSON-RPC for 100x faster syncing of historical data. HyperSync is used by default in Envio HyperIndex, with the use of RPC being optional. Using HyperSync, application developers do not need to worry about RPC URLs, rate-limiting, or managing infrastructure and can easily sync large datasets in a few minutes, something that would usually take hours or days via RPC.
+
+HyperSync is also available as a standalone API for data analytic use cases. Data analysts can interact with the HyperSync API using JavaScript, Python, or Rust clients and extract data in JSON, Arrow, or Parquet formats. For more information, visit the HyperSync documentation [here](https://docs.envio.dev/docs/overview-hypersync).
+
+
+## HyperIndex Key Features
```
Comment (kmurphypolygon) [inline docs/tools/data/envio.md]:
```suggestion
## HyperIndex key features 
```

File: docs/tools/data/envio.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+HyperSync is also available as a standalone API for data analytic use cases. Data analysts can interact with the HyperSync API using JavaScript, Python, or Rust clients and extract data in JSON, Arrow, or Parquet formats. For more information, visit the HyperSync documentation [here](https://docs.envio.dev/docs/overview-hypersync).
+
+
+## HyperIndex Key Features 
+
+- Contract Import: Autogenerate the key boilerplate for an entire Indexer project off a single or multiple smart contracts. Deploy within minutes. 
+
+- Multi-chain Support: Aggregate data across multiple networks into a single database. Query all your data with a unified GraphQL API. 
+
+- Asynchronous Mode: Fetch data from off-chain storage such as IPFS, or contract state (e.g. smart contract view functions).
```
Comment (kmurphypolygon) [inline docs/tools/data/envio.md]:
```suggestion
- Asynchronous mode: Fetch data from off-chain storage such as IPFS, or contract state (e.g. smart contract view functions).
```

File: docs/tools/data/envio.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+  ethereum-mainnet
+  arbitrum-one
+  arbitrum-nova
+> polygon
+  gnosis
+v optimism
+[↑↓ to move, enter to select, type to filter]
+```
+
+#### Enter in the address of the contract to import
```
Comment (kmurphypolygon) [inline docs/tools/data/envio.md]:
```suggestion
#### Enter the address of the contract to import
```

**Comment by SvenMuller95** [comment]
Hey @kmurphypolygon, thank you for the review and the feedback. I have processed the feedback, so should be all good now. Thanks!

**Review by kmurphypolygon — APPROVED** [review]
LGTM

**Comment by kmurphypolygon** [comment]
> Hey @kmurphypolygon, thank you for the review and the feedback. I have processed the feedback, so should be all good now. Thanks!

Thank you @SvenMuller95 publishing today...

**Pull request merged**